### PR TITLE
Feature/branching page

### PR DESCRIPTION
**Pull request by Meazajao**
Some grid and Color changes

### CODE REVIEW

File: websites-css/branching.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .content-section {
-    background-color: var(--color-searchbar);
-    padding: 40px 10vh;
+    background-color: var(--color-content-section);
+    padding: 30px;
     color: white;
```
Comment (Endorze) [inline websites-css/branching.css]: Use the global variables, i think we made one called --color-bg-text that gives white color 😄

**Review by Endorze — APPROVED** [review]
Great job! 🌟

**Review by Endorze — CHANGES_REQUESTED** [review]
You need to fix the merge conflicts and then push again.

Do like this:
1. `git pull origin main`
2. fix merge conflicts, then commit
3. `git push`

**Review by Endorze — CHANGES_REQUESTED** [review]
You're currently 19 commits behind from the Main repository, you need to make sure you're in your branch and do git pull origin main so that you get all current code, then make sure you do git push so the branch on github has all the changes from main.

**Review by Endorze — APPROVED** [review]
Skiter det sig så skiter det sig

**Review by Lvhacker1 — APPROVED** [review]

**Pull request merged**